docs/framework/wcf/extending/understanding-state-changes.md

```diff
@@ -22,12 +22,12 @@ This topic discusses the states and transitions that channels have, the types us
   
  Every <xref:System.ServiceModel.ICommunicationObject> starts out in the Created state. In this state, an application can configure the object by setting its properties. Once an object is in a state other than Created, it is considered immutable.  
   
- ![Channel state transition](./media/channelstatetranitionshighleveldiagram.gif "ChannelStateTranitionsHighLevelDiagram")  
+ ![Dataflow diagram of the channel state transition.](./media/understanding-state-changes/channel-state-tranitions-high-level-diagram.gif "ChannelStateTranitionsHighLevelDiagram")  
```

Comment:
```suggestion
 ![Dataflow diagram of the channel state transition.](./media/understanding-state-changes/channel-state-tranitions-high-level-diagram.gif)  
```

Reply:
There's a typo on the file name: tranitions

image name could also be simplified to just channel-state-transitions.gif